packages/react-native-performance/src/event-emitter.ts

```diff
@@ -0,0 +1,27 @@
+import { PerformanceEntry } from "./performance-entry";
+
+type Callback = (entry: PerformanceEntry) => void;
+
+export const createEventEmitter = () => {
+  const callbacks = new Set<Callback>();
+
+  const addEventListener = (callback: Callback) => {
+    callbacks.add(callback);
+  };
+
+  const removeEventListener = (callback: Callback) => {
+    callbacks.delete(callback);
+  };
+
+  const emit = (event: PerformanceEntry) => {
```

Comment:
I think the event type should be a generic passed when calling `createEventEmitter`

Reply:
I didn't understand the comment. Could you elaborate?

Reply:
this function does not really care about the type of the event, so IMHO it should be a generic type instead.

Reply:
got it